src/jupyter_contrib_nbextensions/nbextensions/execution_dependencies/README.md

```diff
@@ -0,0 +1,38 @@
+execution_dependencies
+======================
+
+Writing extensive notebooks can become very complicated since many cells act as stepping stones to produce intermediate results for later cells. Thus, it becomes tedious to
+keep track of the cells that have to be run in order to run a certain cell. This extension simplifies handling the execution dependencies by introducing tag annotations to
+identify each cell and indicate a dependency on others. This improves on the current state which requires remembering all dependencies by heart or annotating the cells in the comments.
```

Comment:
might be worth noting that dependencies are *definitely* executed, rather than say only being executed once per kernel session. This may be important for cells which take a long time to execute...

Reply:
True, I should add that. Thanks for reviewing my code, I am very grateful for that! I really like jupyter notebooks and I am very interested in contributing more extensions in the future.

Reply:
No problem, happy to help :smile: And of course, we'd be happy to have anything you think might be useful to include here :wink: